habitat_baselines/config/default.py

```diff
@@ -59,6 +59,16 @@
 _C.RL.POLICY = CN()
 _C.RL.POLICY.name = "PointNavBaselinePolicy"
 # -----------------------------------------------------------------------------
+# OBS_TRANSFORMS CONFIG
+# -----------------------------------------------------------------------------
+_C.RL.POLICY.OBS_TRANSFORMS = CN()
+_C.RL.POLICY.OBS_TRANSFORMS.active = tuple()
```

Comment:
Maybe more explicit and conventional:
```suggestion
_C.RL.POLICY.OBS_TRANSFORMS.ENABLED_TRANSFORMS = tuple()
```

Reply:
@mathfac Should we update POLICY as well? I just wanted to be consistent with our .name and stuff.